crates/compiler/build/src/link.rs

```diff
@@ -219,9 +219,8 @@ pub fn build_zig_host_native(
         // include the zig runtime
         // "-fcompiler-rt", compiler-rt causes segfaults on windows; investigate why
         // include libc
-        "--library",
-        "c",
-        "-dynamic",
+        "-lc",
+        "-rdynamic",
```

Comment:
What is the difference between `-dynamic` and `-rdynamic`?

Reply:
```
  -rdynamic                      Add all symbols to the dynamic symbol table
  -dynamic                       Force output to be dynamically linked
```

we want the first one, this was a typo